Src/MoneyFox.Ui/Views/Settings/SettingsPage.xaml

```diff
@@ -19,6 +19,15 @@
 
                 <Picker ItemsSource="{Binding AvailableCurrencies}"
                         SelectedItem="{Binding SelectedCurrency, Mode=TwoWay}" />
+
+                <Label Style="{StaticResource PageTitle}"
+                       Text="{x:Static resources:Translations.AccountIntroductionTitle}" />
+
+                <Label HorizontalOptions="StartAndExpand"
+                       Text="{x:Static resources:Translations.AccountSettingText}" />
+
+                <Picker ItemsSource="{Binding AvailableAccounts}"
```

Comment:
There is a AccountPicker control. I think you could use that here.

Reply:
I created a separate view model, but I stick to Picker control as AccountPicker control has additional UI elements which are not in line with the settings page.